scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionBinomial.scala

```diff
@@ -55,6 +55,11 @@ trait H2OMOJOPredictionBinomial extends PredictionWithContributions {
         if (getWithLeafNodeAssignments()) {
           resultBuilder += pred.leafNodeAssignments
         }
+        if (getWithStageResults()) {
+          val p0Array = pred.stageProbabilities
+          val p1Array = p0Array.map(1 - _)
```

Comment:
just to understand -> is it for consitency in output with multinomial prediction?

Reply:
yes, you're right. `pred.stageProbabilities` contains just  probabilities for one class. For multinomial, `pred.stageProbabilities` contains probabilities for all classes. I just wanted provide a similar output for both type of problems.

Reply:
Yup, like it. Thanks for the explanation!